website/src/components/Buttons/LikertButtons.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, SimpleGrid } from "@chakra-ui/react";
+import { PropsWithChildren, ReactNode } from "react";
+
+export const LikertButtons = ({
+  isDisabled,
+  options,
+  value,
+  onChange,
+}: PropsWithChildren<{
+  isDisabled: boolean;
```

Comment:
is there a situation where we want to disable the buttons?

Reply:
They are disabled when in review mode, in review mode we disable all the inputs.